commune_app/templates/base.html

```diff
@@ -24,10 +24,28 @@
         </span></button>
         <div class="collapse navbar-collapse" id="navbarResponsive">
             <ul class="navbar-nav ml-auto">
-            <li class="nav-item"><a href="{% url 'Main_Page' %}" class="nav-link active text-white">Home</a></li>
+            <li class="nav-item"><a href="{% url 'main_page' %}" class="nav-link active text-white">Home</a></li>
             <li class="nav-item"><a href="#story" class="nav-link active text-white">About Us</a></li>
 			<li class="nav-item"><a href="{% url 'signup' %}" class="nav-link active text-white">Join Us</a></li>
-            <li class="nav-item"><a href="{% url 'commune' %}" class="nav-link active text-white">Communes</a></li></ul>
+            <li class="nav-item"><a href="{% url 'commune' %}" class="nav-link active text-white">Communes</a></li>
+            {% if request.user.is_authenticated %}
+                <li class="nav-item"><a href="{% url 'logout' %}" class="nav-link active text-white">Logout {{ request.user.username }}</a></li>
+            {% else %}
+                    <li class="nav-item"><a href="{% url 'logout' %}" class="nav-link active text-white">Logout</a></li>
```

Comment:
if the user is NOT authenticated 
I think you can remove this line - you don't need logout when the user is not logged in
```suggestion
```